temboardui/plugins/statements/__init__.py

```diff
@@ -722,7 +722,9 @@ def pull_data_for_instance(app, session, instance):
         )
         add_statement(session, instance, json_decode(body))
     except Exception as e:
-        logger.error('Could not get statements data')
+        error = 'Error while fetching statements from instance: ' + \
+            json.loads(e.read())['error']
```

Comment:
Though not new from this commit, this `e.read()` may crash because not all `Exception` types have this method; I'd suggest catching a more accurate class (i.e. one with a `read()` method).

Reply:
I see this is addressed in #836, fine then.